### ISSUE TITLE
Construct new LLNL ESGF-1.5 search index

### ISSUE THREAD

**Issue by lliming** [esgf2-us/esgf-1.5-design#16]
1. Create a new Argonne ESGF-1.5 index
2. Populate the next index by converting the LLNL Solr index entries to Globus Search entries.

**Comment by lliming** [comment]
Currently blocked by #6

**Comment by jpnavarro** [comment]
New plan is to implement a single ESGF2-US wide search index
